### ISSUE TITLE
Chat box - Scroll to bottom arrow accessibility issue

### ISSUE THREAD

**Issue by lotheren** [carbon-design-system/carbon-ai-chat#1332]
### Browser

Chrome

### Operating system

MacOS

### Package version

1.12.0

### React version (if applicable)

18.3.1

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

Ran the accessibility checker on when the down arrow was present and got the following violation: 

<img width="176" height="174" alt="Image" src="https://github.com/user-attachments/assets/215d407f-ca79-4dab-ac9a-37e902a63b86" />

<img width="1020" height="134" alt="Image" src="https://github.com/user-attachments/assets/371991cf-8e12-4f29-b116-23f8d3ee31c9" />

### Reproduction / example

https://chat.carbondesignsystem.com/tag/latest/demo/index.html

### Steps to reproduce

In the chat box have enough text that the down arrow is present when the user is scrolled back up to the top/middle.

<img width="379" height="202" alt="Image" src="https://github.com/user-attachments/assets/1c2302ed-3baa-4ae5-9104-acfbb8aea4a5" />

Can also be seen here: 
https://chat.carbondesignsystem.com/tag/latest/demo/index.html
<img width="1114" height="992" alt="Image" src="https://github.com/user-attachments/assets/7bfc5659-628d-4377-9312-a5b9a59d164b" />

cc: @davidmenendez 

### Suggested severity

None

### Code of conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-ai-chat/issues) for duplicate problems

**Comment by anish-mm-05** [comment]
@davidmenendez 
Seems like the merged pr is present on the latest build but still there are violations present on the latest demo page itself.
https://chat.carbondesignsystem.com/tag/latest/demo/index.html

<img width="1711" height="872" alt="Image" src="https://github.com/user-attachments/assets/100ddcc4-77b3-4360-b7fa-3c20b6f49408" />

**Comment by davidmenendez** [comment]
hey @anish-mm-05 this should be resolved now. i double checked the latest version of the demo you linked above and there is no violation. i'm going to mark this as complete. if you run into any other issues feel free to open this back up or reach out directly!
